web/src/app/rotations/RotationUserList.js

```diff
@@ -36,6 +38,7 @@ const mutation = gql`
   }
 `
 
+@withStyles(styles)
```

Comment:
Instead of adding everything, just pull the pieces needed. This will result in dumping all of material styles on the page instead of only `cardHeader`